src/app/modules/shared_models/user_item.nim

```diff
@@ -105,7 +105,8 @@ proc setup*(self: UserItem,
   # Setup emojiHash:
   if pubKey == "" or not singletonInstance.utils.isChatKey(pubKey):
     self.emojiHash = ""
-  self.emojiHash = singletonInstance.utils.getEmojiHashAsJson(pubKey)
+  else:
+    self.emojiHash = singletonInstance.utils.getEmojiHashAsJson(pubKey)
```

Comment:
We had a useless `if` before. Now it's fixed. Again not the source, just something I found